vic/drivers/shared_image/src/vic_mpi_support.c

```diff
@@ -1671,6 +1454,181 @@ create_MPI_param_struct_type(MPI_Datatype *mpi_type)
 }
 
 /******************************************************************************
+ * @brief   Create an MPI_Datatype that represents the alarm_struct
+ * @details This allows MPI operations in which the entire alarm_struct
+ *          can be treated as an MPI_Datatype.
+ * @param mpi_type MPI_Datatype that can be used in MPI operations
+ *****************************************************************************/
+void
+create_MPI_dmy_struct_type(MPI_Datatype *mpi_type)
+{
+    int           nitems; // number of elements in struct
+    int           status;
+    int          *blocklengths;
+    size_t        i;
+    MPI_Aint     *offsets;
+    MPI_Datatype *mpi_types;
+
+    // nitems has to equal the number of elements in global_param_struct
+    nitems = 5;
+    blocklengths = malloc(nitems * sizeof(*blocklengths));
+    if (blocklengths == NULL) {
+        log_err("Memory allocation error in create_MPI_dmy_struct_type().")
```

Comment:
Below you do not include the function name. Standardize. If the function name comes from `log_err` then we don't need it and you can delete it here (and probably everywhere else in the code  -- just open a cleanup issue for that).  If it is not provided by `log_err` then you need to include it in l.1561 and following as well.